experimental/traffic-portal/src/app/api/statuses.service.ts

```diff
@@ -0,0 +1,71 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { APIService } from './base-api.service';
+
+export interface StatusesModel {
+	description?: string;
+	id?: number;
+	lastUpdated?: Date;
+	name?: string;
+}
```

Comment:
Instead of modeling this yourself, just import the relevant types from the `trafficops-types` package